scripts/install

```diff
@@ -107,6 +133,10 @@ ynh_replace_string --match_string="<BaseUrl />" --replace_string="<BaseUrl>$path
 # Disable Setup Wizard
 ynh_replace_string --match_string="<IsStartupWizardCompleted>false</IsStartupWizardCompleted>" --replace_string="<IsStartupWizardCompleted>true</IsStartupWizardCompleted>" --target_file="$config_path/system.xml"
 
+# Lower logging verbosity
+cp "$config_path/logging.default.json" "$config_path/logging.json"
```

Comment:
I have this error message during upgrade from master
> Warning: cp: cannot stat '/var/lib/jellyfin/logging.default.json': No such file or directory

Reply:
Resolved in upgrade script. It should be OK in install script. ;)